packages/pn-personafisica-webapp/public/locales/en/notifiche.json

```diff
@@ -70,7 +70,7 @@
     "disclaimer-link": "Find out more",
     "payment": {
       "title": "Payment",
-      "subtitle": "In this notification, there are multiple payment alerts: select the one you want to pay. Some alerts include notification fees.",
+      "subtitle": "In this notification, there are multiple payment notices: select the one you want to pay. Some alerts include notification fees.",
```

Comment:
@AliceAzzolini immagino che debba cambiare anche il seguente "alerts"?
"In this notification, there are multiple payment notices: select the one you want to pay. Some **alerts**->**notices** include notification fees.

@NicolaGiornetta

Reply:
Yup, see below:

```suggestion
      "subtitle": "In this notification, there are multiple payment notices: select the one you want to pay. Some notices include notification fees.",
```